app/assets/stylesheets/application.css.scss.erb

```diff
@@ -175,8 +175,9 @@ td, th {
     background-color: #F7F7F7;
 }
 .our-search-btn{
-  background-color: #EED8AE !important;
-  color: #000 !important;
+  background-color: #00897b !important;
+  color: #fff
```

Comment:
Is it necessary to make use of 'important'? In addition, I don't think lines 179 and 180 should be on different lines